### PR TITLE
Invalidate python CircleCI cache to pull in mkdocs fix

### PR DESCRIPTION
**Pull request by consoull**
### What change does this PR introduce and why?
Invalidates python CircleCI cache to pull in mkdocs fix for google fonts API change https://github.com/squidfunk/mkdocs-material/issues/6983.

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 def run(args: Namespace) -> None:
     df_dataset = pd.read_csv(args.dataset_csv, storage_options={"anon": True}, converters={"captions": pd.eval})
     df_dataset["mask"] = df_dataset["mask"].apply(lambda mask: SegmentationMask(locator=mask, labels={1: "PERSON"}))
-    upload_dataset(args.dataset_name, df_dataset)
+    upload_dataset(args.dataset_name, df_dataset, id_fields=["locator"])
```
Comment (consoull) [inline examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: Needed to update to fix a separate breakage caused by https://github.com/kolenaIO/kolena/pull/546

Comment (gordonhart) [reply examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: Before we land this — should we update the behavior to prefer `locator` when present?

Comment (consoull) [reply examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: I would rather scope this change down to fixing the CI. Changing the `id_field` inference order is not backwards compatible
cc: @munkyshi

Comment (gordonhart) [reply examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: Agreed, but from my perspective, we already broke it by changing it to prefer `*_id` or `id_*` over `locator`.

Comment (munkyshi) [reply examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: Ugh, very true -- I hadn't really considered the breakage, sorry.

IMO preferring `*_id` is still the ideal end behavior (I'd prefer to go with an explicitly specified ID over assuming the locator uniquely qualifies a datapoint). What's the ID field that's taking precedence over `locator` in this case?

Comment (munkyshi) [reply examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py:28]: Oh, I see -- the results are keyed by `locator` so even though `image_id` is unique, it doesn't work for results resolution.

IMO I'd rather be explicit about `locator` -- open to counterarguments

**Review by munkyshi — APPROVED** [review]

**Pull request merged**